README.md

```diff
@@ -101,14 +101,14 @@ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ```
 The following template variables need to be replaced as follows with values from your vSphere environment:
 * *\<Windows_VM_template\>*: template name
-* *\<VM Network Name\>*: network name
+* *\<VM Network Name\>*: network name, must match the network name where other Linux workers are in the cluster
```

Comment:
I just realized we need to call this out for all MachineSets. And yes, in this PR and not a follow up one. Just reframe the PR and make it a generic update.

Reply:
makes sense

Reply:
Looking more closely, we seem to have explained this correctly for other `MachineSets`. However for Azure for `image`, we should just say `<image> should be a WindowsServer image offering that defines the 2019-Datacenter-with-Containers SKU`. All verbiage regarding the issue we ran into before needs to be removed. The Azure change however has been reflected in the OpenShift docs so @EricPonvelle does not need to update anything.

Reply:
/cc @EricPonvelle